tests/func/test_import.py

```diff
@@ -75,6 +75,26 @@ def test_import_dir(git, dvc_repo, erepo):
     assert git.git.check_ignore(dst)
 
 
+def test_import_non_cached(git, dvc_repo, erepo):
+    src = "non_cached_output"
+    dst = src + "_imported"
+
+    erepo.dvc.run(
+        cmd="echo hello > %s" % src, outs_no_cache=[src], cwd=erepo.root_dir
```

Comment:
this formatting looks inconsistent with `test_run.py` but that's what the pre-commit hooks did.

Reply:
Btw, we don't use `%` to format strings. Please use `format()` instead.